platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/MigrationZipFile.java

```diff
@@ -145,6 +155,22 @@ public boolean isClosed() {
     return closed;
   }
 
+  /**
+   * Closes the open {@link ZipFile} and deletes the file, key, and checksum
+   *
+   * @return
+   */
+  public boolean deleteQuitetly() {
+    try {
+      close();
+    } catch (IOException e) {
+      throw new MigrationException("Error closing zip file [%s] [%s].", zipPath, e);
+    }
+    return (FileUtils.deleteQuietly(zipPath.toFile()))
```

Comment:
❗️ Might want consider to use `&` instead of `&&` to make sure we attempt to delete all 3 before bailing out.